lib/LLVMPasses/LLVMARCOpts.cpp

```diff
@@ -142,8 +142,8 @@ static bool canonicalizeInputFunction(Function &F, ARCEntryPointBuilder &B,
         // Have not encountered a strong retain/release. keep it in the
         // unknown retain/release list for now. It might get replaced
         // later.
-        if (NativeRefs.find(ArgVal) == NativeRefs.end()) {
-           UnknownObjectRetains[ArgVal].push_back(&CI);
+        if (!NativeRefs.contains(ArgVal)) {
```

Comment:
Seems the build is failing because`llvm::DenseSet` doesn't have a count method, @gottesmm we normally use [`count`](http://llvm.org/doxygen/classllvm_1_1detail_1_1DenseSetImpl.html#afe504aa31a6a354cec13f5b32d0b1d9d) so perhaps we could use this here as well? :)

Reply:
It seems like the change to `llvm::DenseSet` hasn't landed on `swift/main` branch so it's causing this issue..? The `contains` method exists on llvm-project's `apple/stable/20210107` branch; I'm not sure how the CI works but could this be fixed by moving the base branch? :)

Reply:
Yeah, seems like it hasn't ... I would say that use `count` instead for `Set` like would just be fine and already an improvement, but since the whole point of the SR is to use `contains` I'm not really sure about that, so cc @varungandhi-apple @gottesmm